pyvista/core/dataobject.py

```diff
@@ -50,7 +50,7 @@ class DataObject:
 
     def __init__(self, *args, **kwargs) -> None:
         """Initialize the data object."""
-        super().__init__()
+        # super().__init__(*args, **kwargs)
```

Comment:
Is this a tricky case where we seemingly never have non-`object` `super()` so we'd need to call `super().__init__()`, except in multiple inheritance we might end up with a `super()` that's a, unrelated class, despite _this_ class having no parent class?

Reply:
I think so, I was struggling to capture exactly why this was needed but I think that sums it up

Reply:
I wonder if it would be worth doing something like
```python
try:
    super().__init__(*args, **kwargs)
except TypeError:
    # super() maps to object
    super().__init__()
```
Just making sure that if someone (potentially downstream from PyVista) tries to inherit from us, we're also playing nicely.

Reply:
Good call, seems to work!